### PR TITLE
Issue/2572 - Use ! empty()

### PR DESCRIPTION
**Pull request by amdrew**
#2572

### CODE REVIEW

File: includes/integrations/class-edd.php [hunk 1]
```diff
@@ -456,7 +456,7 @@ public function get_referral_description( $payment_id = 0 ) {
 
 			$desc = get_the_title( $item['id'] );
 
-			if ( isset( $item['options']['price_id'] ) && null !== $item['options']['price_id'] ) {
+			if ( ! empty( $item['options']['price_id'] ) ) {
```
Comment (pippinsplugins) [inline includes/integrations/class-edd.php]: This will fail for older products that have a price ID of `0`.

Comment (amdrew) [reply includes/integrations/class-edd.php]: Ahh yes, that's right, they used to be indexed at `0`. Got it

**Comment by amdrew** [comment]
@pippinsplugins See latest commit, happy with that?
https://github.com/AffiliateWP/AffiliateWP/pull/2573/commits/eab072eee5d6018f28faceda5449922475824205

It seems like checking if a product is actually variable priced is a better option than checking against the `price_id`. Method is supported since EDD v2.2, customers should upgrade if on an earlier version of EDD.

**Comment by pippinsplugins** [comment]
Yeah that's great, I like that.

**Pull request merged**